MetaMask/metamask-extension
# Whats new blockaid popup

## Explanation

Let's communicate to users this amazing new feature that we have!! 🎉

Acceptance Criterias

A new message with image + text should be added to our existing What's new modal
Clicking on Enable security alerts should take the user to this feature within Settings (and potentially also turn the feature on - TBD)
This should be behind the same feature/build flag that we have for the rest of Blockaid integration feature

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/927

## Screenshots/Screencaps

### Before
![oldwn](https://github.com/MetaMask/metamask-extension/assets/44811/53f93257-37e1-431e-b4a6-2e21f0f0e02f)


### After
<img width="485" alt="Screenshot 2023-07-25 at 18 01 32" src="https://github.com/MetaMask/metamask-extension/assets/44811/8daa6469-2435-497a-a205-031baa0fc07e">


## Manual Testing Steps

- install MM afresh
- Register/Import wallet
- What's new popup should be shown and blockaid should be second what's new on the popup after swaps what's new
## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
